include/io.h

```diff
@@ -40,7 +40,7 @@ class IO {
 
   //! Write stresses
   void write_stresses(
-      const std::vector<std::array<double, Tdim * 2>>& stresses);
```

Comment:
In the next push, I will still have `materialpoints` as the input. Please let me know how to change them